examples/passage_reranker/ms_marco_evaluator.py

```diff
@@ -0,0 +1,39 @@
+from texar.torch import HParams
+
+from forte.common import Evaluator, Resources
+from forte.data.multi_pack import MultiPack
+from forte.data.ontology import Query
+
+from eval_script import compute_metrics_from_files
+
+
+class MSMarcoEvaluator(Evaluator[MultiPack]):
+
+    def __init__(self):
+        super().__init__()
+        self.pred_packs = []
+        self.ref_packs = []
+        self._score = None
+
+    # pylint: disable=attribute-defined-outside-init
+    def initialize(self, resource: Resources, configs: HParams):
+        self.resource = resource
+        self.config = configs
+
+    def consume_next(self, pred_pack, _):
+        self.pred_packs.append(pred_pack)
```

Comment:
It would be better to store only some necessary information from the pack, here we only need the ``doc_id``?